docs/api-guide/schemas.md

```diff
@@ -215,6 +215,110 @@ This also applies to extra actions for `ViewSet`s:
 If you wish to provide a base `AutoSchema` subclass to be used throughout your
 project you may adjust `settings.DEFAULT_SCHEMA_CLASS`  appropriately.
 
+
+### Grouping Operations With Tags
+
+Tags can be used to group logical operations. Each tag name in the list MUST be unique. 
+
+---
+**Django REST Framework generates tags automatically with following logic:**
+1. Extract tag from `ViewSet`. 
+    1. If `ViewSet` name ends with `ViewSet`, or `View`; remove it.
+    2. Convert class name into lowercase words & join each word using a space. 
+    
+    Examples:
+    
+        ViewSet Class   |   Tags
+        ----------------|------------
+        User            |   ['user']	 
+        UserView        |   ['user']	 
+        UserViewSet     |   ['user']	
+        PascalCaseXYZ   |   ['pascal case xyz']
+        IPAddressView   |   ['ip address']
+
+2. If View is not an instance of ViewSet, tag name will be first element from the path. Also, any `-` or `_` in path name will be replaced by a space.
```

Comment:
I wonder if we ought to just use the path style as the default case?

I think that'd be more obvious, more consistent across a code base, and a little more simple.

Reply:
Sure. I will implement the path style as the default case.

Reply:
Yes, I think the default implementation should use the path. Folks can pass tags, or subclass for something cleverer.